src/components/Tokens/TokenTable/TokenRow.tsx

```diff
@@ -543,7 +561,20 @@ export default function LoadedRow({
         volume={<ClickableContent>{volume ? formatDollarAmount(volume ?? undefined) : '-'}</ClickableContent>}
         sparkLine={
           <SparkLine>
-            <ParentSize>{({ width, height }) => <SparklineChart width={width} height={height} />}</ParentSize>
+            <ParentSize>
+              {({ width, height }) => (
+                <LineChart
+                  data={pricePoints}
+                  getX={(p: PricePoint) => widthScale(p.timestamp)}
+                  getY={(p: PricePoint) => rdScale(p.value)}
+                  curve={curveCardinal.tension(curveTension)}
+                  color={pricePercentChange && pricePercentChange < 0 ? theme.accentFailure : theme.accentSuccess}
+                  strokeWidth={2}
```

Comment:
Can we try 1.5px with here? I think that's how figma is

Reply:
done ty